extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -284,6 +336,19 @@ export abstract class DialogBase<DialogResult> {
 		});
 	}
 
+	protected getDataForDeclarativeTableList<T>(
+		allItems: T[],
+		selectedItems: T[],
+		enabledStateGetter: TableListItemEnabledStateGetter<T> = DefaultTableListItemEnabledStateGetter,
+		rowValueGetter: TableListItemValueGetter<T> = DefaultTableListItemValueGetter,
+		itemComparer: TableListItemComparer<T> = DefaultTableListItemComparer): any[][] {
+		return allItems.map(item => {
+			const idx = selectedItems.findIndex(i => itemComparer(i, item));
+			const stateColumnValue = { value: idx !== -1, enabled: enabledStateGetter(item), style: tableRow, ariaLabel: item };
```

Comment:
I also added aria labels for the checkboxes here, since that's usually something that the accessibility testers ask for so that users know which checkbox was checked/unchecked